versions-dev.json

```diff
@@ -0,0 +1,115 @@
+{
+    "latest": {
+      "SANDS": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_SANDS.git"
+      },
+      "chemicals": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_chemicals.git"
+      },
+      "computation": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_computation.git"
+      },
+      "controlledTerms": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_controlledTerms.git"
+      },
+      "core": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_core.git"
+      },
+      "ephys": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_ephys.git"
+      },
+      "publications": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_publications.git"
+      },
+      "specimenPrep": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_specimenPrep.git"
+      },
+      "stimulation": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_stimulation.git"
+      },
+      "neuroimaging": {
+        "repository": "https://github.com/openMetadataInitiative/openMINDS_neuroimaging"
+      }
+    },
+    "v1.0": {
```

Comment:
Not sure if we actually want to keep the stable versions in the versions-dev.json since it would mean that all of the elements would be built twice (once for versions.json and once for versions-dev.json). However, it might be valuable to remove the fixed commits as well to have the same structure in the "development" branch as in the "main" branch but always built on the latest head. we could even call it v1.latest then.